CHANGELOG.md

```diff
@@ -1,35 +1,136 @@
+    - "Internet-required data/Satellite orthophoto mosaic (10-0.2m)"
+- Add new "Internet-required data/Blue Marble shaded relief and Bathymetry
+  (500m)" WMS layer from NASA Global Imagery Browse Services (GIBS)
+- Add new "Places/" layers from Asiaq/NunaGIS:
+    - Buildings
+    - Roads
+- Add new "Biology/Vegetation/Vegetation classification map (1km)" layer from
+  the Raster Circumpolar Arctic Vegetation Map produced by Raynolds et al.,
+  2019.
+
+
+### Updated
+
+- Fix descriptions of "Hydrology" outlets and basins layers.
+- Rename "Places/Place names database" -> "Places/Points of interest". Towns and
+  settlements are now excluded from this layer (in favor of the more-up-to-date
+  "Towns" and "Settlements" layers.
+- Update the BedMachine dataset ("Terrain models/BedMachine") to v5, which was
+  released in September 2022.
+- Update the geothermal heat flow dataset ("Geophysics/Heat flow (Colgan et
+  al.)" layers) to v2.
+- Update "Sea ice/Weekly age (12.5km)" layers (2010-2020 -> 2011-2021)
+- Update "Sea ice/Monthly mean concentration (25km)" layers (2010-2021 -> 2011-2022)
+- Update the "Oceanography/Seawater temperature 2018 (25km)" layers:
+    - Use the World Ocean Atlas 2023 Temperature data. These new layers are
+      climatological averages between 1991-2020. Previous input data (WOA 2018)
+      was an average for the years 1955-2017.
+    - Update layer group name to remove "2018" ("Oceanography/Seawater Temperature
+      2018 (25km)" -> "Oceanography/Seawater Temperature (25km))"
+    - Improve layer descriptions
+    - Expand layer extent to QGreenland 'background' boundary.
+- Update the "Glaciology/Surface elevation change" layers to the latest version
+  (v3) of the Climate Change Initiative (CCI) Surface Elevation Change dataset
+  from the European Space Agency (ESA). This extends the timeseries to include a
+  surface elevation change layer for 2016-2020.
+- Rename "Geophysics/World Magnetic Model/Dip poles" layer group ->
+  "Geophysics/World Magnetic Model/Geomagnetic north pole"
+- Rename "Sea ice/Median extent" layer group -> "Sea ice/Median extent
+  (1981-2010)"
+- Update "Oceanography/Bathymetry/Depth (400m)" layer to use the General
+  Bathymetric Chart of the Oceans (GEBCO) Grid 2023, replacing the Bathymetric
+  Chart of the Arctic Ocean (IBCAO) dataset. The GEBCO grid includes the IBCAO
+  grid. This update includes the most recent data and expands the spatial extent
+  of this layer to the QGreenland background boundary to make it consistent with
+  other ocenaography layers.
+- Replace "Human Activity/Research sites/" layers related to PROMICE and GC-Net
+  ("PROMICE automated weather stations", "Former PROMICE automated weather
+  stations", "GC-Net automated weather stations") with one layer ("PROMICE and
+  GC-Net automated weather stations") from a more up-to-date dataset provided by
+  GEUS.
+- Fix title of BedMachine "ice thickness error" layer (remove `/`)
```

Comment:
Maybe move this up to be next to the other note about updating to BedMachine v5?

Reply:
b3cc775e